Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -4022,13 +4033,24 @@ public async Task<bool> TryLogWarningAsync(SandboxedProcessOutput standardError,
                 return false;
             }
 
-            FormatOutputAndPaths(
+            bool errorWasTruncated = false;           
+            // Ignore empty lines
+            var standardErrorInResult = await standardError.ReadValueAsync();
+            var standardOutputInResult = await standardOutput.ReadValueAsync();
+            if (warningsError.Replace(Environment.NewLine, string.Empty).Trim().Length != standardErrorInResult.Replace(Environment.NewLine, string.Empty).Trim().Length ||
+                warningsOutput.Replace(Environment.NewLine, string.Empty).Trim().Length != standardOutputInResult.Replace(Environment.NewLine, string.Empty).Trim().Length)
+            {
+                errorWasTruncated = true;
+            }
+
+                FormatOutputAndPaths(
                 warningsOutput,
                 warningsError,
                 standardOutput.FileName,
                 standardError.FileName,
                 out string outputTolog,
-                out string outputPathsToLog);
+                out string outputPathsToLog,
+                errorWasTruncated);

```

Comment:
>errorWasTruncated [](start = 16, length = 17)

Do we even allow truncating warnings? There wouldn't be any backing file with just the filtered warnings to refer to.

Reply:
As to the test case, I do get the errorWasTruncated == true.
So I guess that is the case.